### PR TITLE
214605 apply mix format

### PR DESCRIPTION
**Pull request by aMasakiTakahashi**
https://acsmine.tok.access-company.com/redmine/issues/214605

### CODE REVIEW

**Review by aDaichiOta — APPROVED** [review]

File: .formatter.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  inputs: ["{mix,.formatter}.exs", "{core,eal,lib,local,test}/**/*.{ex,exs}"],
+  locals_without_parens: [defun: 2, defunp: 2]
```
Comment (ttokit) [inline .formatter.exs:3]: pls add [`defunpt/2`](https://hexdocs.pm/croma/Croma.Defun.html#defunpt/2) and [`defpt/2`](https://hexdocs.pm/croma/Croma.Defpt.html#defpt/2)

Comment (aMasakiTakahashi) [reply .formatter.exs]: Thank you. I fixed it.
1cace48cb5383ec8d22948b3331d87667c87f4fb

Currently, there are no functions defined with 'defunpt' or 'defpt' on a single line.
So this setting has no effect on the result of 'mix apply'.
However, 'defunpt' and 'defpt' are in frequent use and may one day be used to define a function on a single line.
For that reason, I decided beforehand to treat these functions as exceptions.

**Review by ttokit — APPROVED** [review]
ok,

**Comment by aMasakiTakahashi** [comment]
I close this PR because I created new PR to resolve the conflict.

**Pull request closed without merging**